androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/Settings.kt

```diff
@@ -23,6 +24,7 @@ import com.profusion.androidenhancedvideoplayer.R
 import com.profusion.androidenhancedvideoplayer.styling.Dimensions
 import kotlinx.coroutines.launch
 
+@Stable
 data class SettingsControlsCustomization(
     val speeds: List<Float> = listOf(0.5f, 0.75f, 1.0f, 1.25f, 1.5f),
```

Comment:
Lists are unstable by default (see this [article](https://medium.com/androiddevelopers/jetpack-compose-stability-explained-79c10db270c8)), so if you mark the class as stable and some place uses `SettingsControlsCustomization` with a mutable list can lead some unexpected behaviors. Consider using [Immutable Collections](https://github.com/Kotlin/kotlinx.collections.immutable) instead.